packages/suite-desktop/package.json

```diff
@@ -64,6 +64,7 @@
             "!node_modules/@sentry/**/build",
             "!node_modules/tiny-secp256k1/**/build",
             "!node_modules/blake-hash/**/build",
+            "!node_modules/bcrypto/**/build",
```

Comment:
this excludes binaries in bcrypto from build and prevent app.asar.unpacked folder

before:
![image](https://user-images.githubusercontent.com/3729633/195284945-a7339d31-b0c3-4b8b-8de1-b3455dc843d2.png)

after:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/3729633/195285097-b92123e5-a920-4b22-bed6-6135a36aa96c.png">